media/vp9-codec-string.js

```diff
@@ -0,0 +1,25 @@
+if (MediaSource.isTypeSupported('video/webm; codecs="vp9"')) {
+  log('> VP9, Profile 0, level 1 (later fields defaulted)');
+}
+
+if (MediaSource.isTypeSupported('video/webm; codecs="vp09.00.10.08"')) {
+  log('> VP9, Profile 0, level 1, bit depth 8 (later fields defaulted)');
+}
+
+if (MediaSource.isTypeSupported('video/webm; codecs="vp09.01.20.08.01"')) {
+  log('> VP9, Profile 0, level 1, bit depth 8, 4:2:0 chroma subsampling ' +
```

Comment:
nit: 4:2:0 chroma subsampling is correct, but could apply to eitehr values of 0 or 1. I realize my test documentation is equally ambiguous. You might help clarify that this is value 4:2:0 variety that includes "colocated with luma (0,0)" ... or similar.